lte/gateway/docker/services/build/Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:20.04 AS builder
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV MAGMA_ROOT=/magma
+ENV ARCH=amd64
+ENV OS_RELEASE=ubuntu20.04
+
+RUN apt-get update \
+  && apt-get install -y python3-pip python3-dev git cmake ninja-build \
+  protobuf-c-compiler protobuf-compiler libprotobuf-dev libyaml-dev \
+  libyaml-cpp-dev wget ruby ruby-dev autoconf libtool pkg-config \
+  libgflags-dev libgoogle-glog-dev libssl-dev libboost-all-dev libghc-double-conversion-dev \
+  libsctp-dev libboost-all-dev libevent-dev libdouble-conversion-dev \
+  libgoogle-glog-dev libgflags-dev libiberty-dev liblz4-dev liblzma-dev \
+  libsctp1 libgcrypt-dev bison libidn11-dev flex libtspi1 libtcmalloc-minimal4 \
+  libgoogle-perftools4 sudo libjemalloc2 libsnappy1v5 nlohmann-json3-dev libconfig9 \
+  libconfig-dev libxml2-dev libczmq-dev
+
+RUN gem install fpm
+
+WORKDIR /
+
+RUN git clone https://github.com/magma/magma.git && \
+    cd magma && \
+    git checkout master # TODO, how do we handle versioning for this?
```

Comment:
Perhaps pickup a second argument in the build script that includes version alongside the service that is being built?

Reply:
use argument to provide version , and use master as default option